.github/workflows/ci-build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# AppVeyor and Drone Continuous Integration for MSYS2
+# Author: Renato Silva <br.renatosilva@gmail.com>
+# Author: Qian Hong <fracting@gmail.com>
+
+# Configure
+cd "$(dirname "$0")"
+source 'ci-library.sh'
+mkdir artifacts
+git_config user.email 'ci@msys2.org'
```

Comment:
I do not think it is cool to be using msys2 e-mail here.

Reply:
Yeah, probably not a good idea!